### PR TITLE
first attempt at adding unix support

### PR DESCRIPTION
**Pull request by wolfv**
I don't think this _builds_ successfully yet but it's a first stab at some Linux / UNIX support :)

### CODE REVIEW

File: examples/conda-forge.yaml [hunk 1]
```diff
@@ -1,198 +1,226 @@
 pkg-config:
-  win64: [pkg-config]
+  any: [pkg-config]
```
Comment (seanyen) [inline examples/conda-forge.yaml:2]: nit: Can we use `conda-forge` or just `conda` over `any`?

Ideally we are trying to keep this file in future easy to be integrated with [REP-1111](https://www.ros.org/reps/rep-0111.html).

And for the subkey, I am thinking we can reuse the `conda` [selectors](https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors), for example, `osx`, `win64`, `linux64`, or `armv7l`, etc...

What do you think? 

```yaml
pkg-config:
    conda-forge:
        win64: [pkg-config]
noarch-package:
    conda-forge: [noarch-package]
```

**Comment by wolfv** [comment]
with the latest push I got everything up to RViz to compile (not including RViz, though).
Will try RViz tomorrow -- I already have a patch and know more or less which X11 packages will be required.

Cheers! vinca is working really nicely :)

**Comment by Tobias-Fischer** [comment]
I locally fixed the merge conflicts and I am running a local build. Once it build successful, I will open a new PR.

**Comment by wolfv** [comment]
closed in favor of #3 thanks @Tobias-Fischer !

**Pull request closed without merging**